I3uckwheat/cards-of-carousal
# Refactor HeaderFooterLayout to use global breakpoints

#### 0. Make sure your issue is linked:

Part of #298 

#### 1. Purpose:

Refactor HeaderFooterLayout to use the global breakpoints.


#### 2. Implementation:

Refactor CSS to support using global breakpoints.


#### 3. Possible Issues:

Because of how much I had to change the CSS there is a possibility that some of the styling is now "off" and will look less than optimal at some resolutions.


#### 4. How to Test:


0. Fetch my branch and switch to it
1. Start the application backend then frontend
2. On the welcome screen and join screen verify that the footer looks good at as many different resolutions as you can check it on.


#### 5. Cleanup
